Add a workaround to avoid increased current consumption when forcing PT mode (#285)
NordicSemiconductor/pc-nrfconnect-npm: Changelog.md

## 2.2.3 - Unreleased

### Fixed

-   nPM2100 Decrease Vout below Vbat before forcing PT to avoid increased

>     current consumption.

Reviewer: Please format it so that it starts with "Issue where". I struggle to understand what the issue was here and what was fixed.

Author: does it look better now?

Reviewer: Yes, thanks :)

## 2.2.3 - Unreleased

### Changed

> -   nPM2100:
>     -   Setting the Boost regulator to the forced Pass-through mode will now
>         first set the output voltage VOUT to the minimum setting (1.8 V) as
>         instructed by the datasheet

Reviewer: ```suggestion
-   nPM2100: Setting the Boost regulator to the forced pass-through mode will now first set the output voltage VOUT to the minimum setting (1.8 V), in accordance with the datasheet.
```
